lib/middleware/isAdminUser.js

```diff
@@ -1,19 +1,22 @@
-'use strict';
+module.exports = (req, res, next) => {
+    const licenceAdminList = req.adminUserList.administrators;
+    const currentUserId = req.currentUser.uuid;
 
-function* isAdminUser(req, res, next) {
-  let licenceAdminList = req.adminUserList.administrators;
-  let currentUserId = req.currentUser.uuid;
+    let adminCred;
+    licenceAdminList.every(admin => {
```

Comment:
Similar to my comment for checkLicenceStatus.js, it feels like we should use `forEach()` instead of `every()`.